apps/src/music/blockly/blockUtils.js

```diff
@@ -89,7 +89,8 @@ export function installFunctionBlocks(blockMode) {
       generator
     ) =>
       simple2FunctionCallGenerator(
-        generator.getProcedureName(block.getFieldValue('NAME'))
+        generator.getProcedureName(block.getFieldValue('NAME')),
+        block.id
```

Comment:
Could we simplify some of this by passing block.getProcedureModel().id here? The model is shared between the definition and call blocks.

Reply:
Let's discuss for a potential follow-up!

Reply:
How does https://github.com/code-dot-org/code-dot-org/pull/62359 look?